packages/calcite-components/src/components/panel/panel.scss

```diff
@@ -147,6 +156,26 @@
   margin-inline-end: auto;
   justify-content: center;
 
+  .header-actions {
+    --calcite-action-menu-items-space: var(--calcite-panel-header-action-menu-items-space, 0);
```

Comment:
I don't think the `0` is needed here as its the default in the `action-menu.scss`. right?

My concern is that the `0` within the `action-menu` should be the source of truth. If action-menu changes to something other than 0 then how would we know to update this as well?

Reply:
Applies to other fallback defaults below.

Reply:
I know this is a public prop on Action Menu - I'm not sure the history there (or if needed on that component), but I don't know if this needs to be added here - I think just the color ones are good for this.